statsd_client.ml

```diff
@@ -20,7 +20,6 @@ module Make(U : sig
   (** include Lwt_unix or Unix depending on your use *)
```

Comment:
Should we update these functor comments with Async references? Or remove them entirely?

Reply:
I removed the comment.